src/Microsoft.Deployment.DotNet.Releases/src/ReleaseVersion.cs

```diff
@@ -27,9 +26,9 @@ public class ReleaseVersion : IComparable, IComparable<ReleaseVersion>, IEquatab
             @"(?<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*)";
 
         /// <summary>
-        /// Regular expression for v2 semantic version to capture components into separate groups: major, minor, patch, prerelease and build.
+        /// A regular expression pattern to capture a semantic version (2.0).
         /// </summary>
-        private static readonly string s_semanticVersion2Pattern =
+        public static readonly string SemanticVersionPattern =
```

Comment:
Should the groups be documented?  How is a user supposed to discover these?

Reply:
I'll update the docs.